
<launch>
  <node name="rescue_mission" pkg="wolfdrone" type="takeoff_land.py" output="screen">
    <!-- Location Parameters -->
    <rosparam param = "home_lat">32.265266</rosparam>
    <rosparam param = "home_lon">-111.2736201</rosparam>
    <rosparam param = "land_lat">32.265503</rosparam>
    <rosparam param = "land_lon">-111.274712</rosparam>
      
  </node>
</launch>
